test/controllers/foods_controller_test.rb

```diff
@@ -0,0 +1,50 @@
+require 'test_helper'
+
+class FoodsControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @food = foods(:one)
+  end
+
+  test 'should get index' do
+    get foods_url
+    assert_response :success
+  end
+
+  test 'should get new' do
+    get new_food_url
+    assert_response :success
+  end
+
+  test 'should create food' do
+    assert_difference('Food.count') do
+      post foods_url,
+           params: { food: { measurement_unit: @food.measurement_unit, name: @food.name, price: @food.price } }
+    end
+
+    assert_redirected_to food_url(Food.last)
+  end
+
+  test 'should show food' do
+    get food_url(@food)
+    assert_response :success
+  end
+
+  test 'should get edit' do
+    get edit_food_url(@food)
+    assert_response :success
+  end
+
+  test 'should update food' do
+    patch food_url(@food),
+          params: { food: { measurement_unit: @food.measurement_unit, name: @food.name, price: @food.price } }
+    assert_redirected_to food_url(@food)
+  end
+
+  test 'should destroy food' do
+    assert_difference('Food.count', -1) do
+      delete food_url(@food)
+    end
+
+    assert_redirected_to foods_url
```

Comment:
- [ ] I am not able to run your tests, I can see that you called your folder called `tests` and I don't think that when you are testing with `rspec` should be that way I can't run `rspec spec` and no folder called `spec` you have I suggest that you follow how to generate files and folder when testing with `rspec`
![test](https://user-images.githubusercontent.com/44745855/163470924-686e529c-f3a5-4f43-9605-cad5ea7df600.png)